site/docs/try/releases.md

```diff
@@ -1,5 +1,14 @@
 # Releases
 
+## 0.7.0 Release (June 15, 2021)
+
+* Server side filtering for entries-listing
+* Author & committer improvements in the Python CLI
+* Loosen pins for client install on Python cli
+* Gradle plugin improvements
+* (Development) change to Google Code Style, add spotless plugin
+* (CI) Add OWASP Dependency Check
```

Comment:
```suggestion
* (CI) Add OWASP Dependency Check
* Fix edge case when merging using in memory store
```

Reply:
should we also mention that we fixed this for the Jgit store?

Reply:
yup - added